internal/server.go

```diff
@@ -72,14 +73,15 @@ func (s *Server) Stop() {
 func (s *Server) certManager() *autocert.Manager {
 	client := &acme.Client{DirectoryURL: s.config.ACMEDirectoryURL}
 	binding := s.externalAccountBinding()
+	tlsDomains := strings.Split(s.config.TLSDomain, ",")
```

Comment:
Have you considered doing this in `config.go`? That way you could keep `TLSDomains` in `s.config` as a slice already parsed. Also, it's prolly easier to test there.

If you go ahead with this, you may want to ensure the following still works:

https://github.com/basecamp/thruster/blob/c19ae784d344eae69826f528ae4e27ed65f46dea/internal/server.go#L47

You can see the logged line with:

```bash
cd cmd/thrust
TLS_DOMAIN=example.com go run main.go echo 1
```

Reply:
You could even do some defensive whitespace trimming just in case.